.github/workflows/end-to-end-tests.yml

```diff
@@ -56,3 +69,81 @@ jobs:
           npx nx bundle web-component
           git status
           git diff --word-diff=porcelain --word-diff-regex=... --color | perl -ple 's/^(\x1b[^ -+]{0,6})? (.{81,})$/$1 . " " . substr($2, 0, 40) . " [... " . (length($2)-80) . " bytes ...] " . substr($2, -40)/ex'
+  playwright-tests:
+    name: Run Playwright test-suites
+    timeout-minutes: 60
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        shardIndex: [1, 2, 3, 4]
+        shardTotal: [4]
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-node@v4
+        with:
+          node-version: lts/*
+      - name: Install and run the back-end API, needed for end-to-end testing
+        run: |
+          git clone https://github.com/ReadAlongs/Studio
+          cd Studio
+          pip install -e . -r requirements.api.txt
+          ./run-web-api.sh &
+          # wait for the API to be up
+          curl --retry 20 --retry-delay 1 --retry-all-errors http://localhost:8000/api/v1/langs
+      - name: Install everything
+        run: npm install
+      - name: Install dependencies
+        run: npm ci
+      - name: Run studio-web in the background
+        run: |
+          npx nx build web-component
+          npx nx run-many --targets=serve,serve-fr,serve-es --projects=web-component,studio-web --parallel 6 &
```

Comment:
Unless we actually exercise fr and es, I would drop serve-fr and serve-es from here.

Reply:
We do see `packages/studio-web/tests/studio-web/check-page-1.spec.ts`